packages/shared/lib/sdk/sync.unit.test.ts

```diff
@@ -6,12 +6,45 @@ import configService from '../services/config.service.js';
 import type { CursorPagination, LinkPagination, OffsetPagination } from '../models/Proxy.js';
 import { NangoAction } from './sync.js';
 import { isValidHttpUrl } from '../utils/utils.js';
+import proxyService from '../services/proxy.service.js';
+import type { AxiosResponse } from 'axios';
 
 vi.mock('@nangohq/node', () => {
     const Nango = vi.fn();
     return { Nango };
 });
 
+describe('Proxy', () => {
+    let nangoAction: NangoAction;
+    let nango: Nango;
+    beforeEach(async () => {
+        nangoAction = new NangoAction({
+            secretKey: '***',
+            providerConfigKey: 'github',
+            connectionId: 'connection-1'
+        });
+        nango = new Nango({ secretKey: '***' });
+        (await import('@nangohq/node')).Nango.prototype.getConnection = vi.fn().mockReturnValue({ credentials: {} });
```

Comment:
You can mock `nango` instance directly? this stuff is always so messy

Reply:
I was being consistent with the other tests in this file but I agree it is ugly

Reply:
also nango is a private field of NangoAction so I am not sure I can mock it directly